ui/less/ad_controls.less

```diff
@@ -31,18 +32,15 @@
 
 .shaka-client-side-ad-container {
   .absolute-position();
-}
 
-.shaka-controls-container {
```

Comment:
You need to string this with the server-side-container, so it's applied to both of them.
I think, it's done like this, but please double check with less.js documentation, I haven't worked on less for a while:
```
.shaka-server-side-ad-container .shaka-client-side-ad-container {
  iframe {/* iframe settings here */}
}
```

Reply:
like this
```
.shaka-server-side-ad-container, .shaka-client-side-ad-container {
  iframe {/* iframe settings here */}
}
```
however ,i found that the server-side-ad-container is a flex item inside shaka-controls so it never interfere with the shaka-bottom-controls, so no need to lift it up.
am i missing something?

Reply:
A demonstration of what iam talking about.

![Screenshot from 2021-04-13 01-59-59](https://user-images.githubusercontent.com/56920956/114477138-fdacf800-9bfb-11eb-9667-6c4f78802376.png)
![Screenshot from 2021-04-13 02-00-02](https://user-images.githubusercontent.com/56920956/114477143-00a7e880-9bfc-11eb-8dae-e6f2b081ba7a.png)

i though it should be lifted up like the ```client-side-ad```, but turns out it already flexes with ```bottom-controls-container```

Reply:
Man, I hate UI work((
Okay. I went and checked. We don't have an example of a skippable server-side ad, but looking at the way the IMA SDK adds their UI on the client side, they align it directly over the video element using `position: absolute`. So, essentially, they don't care whether the container is lifted, and we still have to lift the i-frame.

I think it's a safe assumption that they do this with server-side ads as well, so please lift the iframe on the server-side container too.

Reply:
sensible :+1: